Util_Admin.php

```diff
@@ -734,9 +734,9 @@ static public function get_current_page() {
 	 * @return bool
 	 */
 	static public function is_w3tc_admin_page() {
-		if ( isset( $_GET['page'] ) && substr( $_GET['page'], 0, 5 ) == 'w3tc_' )
+		if ( isset( Util_Request::get_string( 'page' ) ) && substr( Util_Request::get_string( 'page' ), 0, 5 ) == 'w3tc_' )
```

Comment:
Use `! empty()`.  You may want to set a variable to avoid calling twice.

Reply:
Changed to ! empty and created a variable to avoid 2 function calls.